src/main/java/com/liveramp/hyperminhash/BetaMinHash.java

```diff
@@ -56,7 +56,7 @@ private BetaMinHash(short[] registers) {
   /**
    * Create a BetaMinHash from the serialized representation returned by {@link #getBytes()}.
    */
-  public static BetaMinHash fromBytes(byte[] bytes) {
+  public static BetaMinHash wrapRegisters(byte[] bytes) {
```

Comment:
This naming seems a bit misleading, since the registers aren't `byte`s. Maybe `fromRegisterBytes`?

Reply:
It looks like this isn't used anywhere, so it might make sense to just remove this method, since the deserialization functionality is in `BetaMinHashSerde`